src/baldr/openlr.cc

```diff
@@ -1,97 +0,0 @@
-#include <cmath>
-#include <string>
-#include <vector>
-
-#include "baldr/graphconstants.h"
-#include "baldr/openlr.h"
-#include "midgard/encoded.h"
-#include "midgard/pointll.h"
-
-#include "proto/trip.pb.h"
-#include "proto/tripcommon.pb.h"
-
-namespace valhalla {
-namespace midgard {
-
-using FormOfWay = OpenLR::LocationReferencePoint::FormOfWay;
-
-FormOfWay road_class_to_fow(const TripLeg::Node& node) {
-  const bool is_missing_rc = !node.has_edge() || !node.edge().has_road_class();
-  if (is_missing_rc || !node.edge().has_use()) {
-    return FormOfWay::OTHER;
-  }
-  const TripLeg::Use use = node.edge().use();
-  const RoadClass rc = node.edge().road_class();
-  const bool one_way =
-      node.edge().has_traversability() && node.edge().traversability() != TripLeg::kBoth;
-  if (node.edge().roundabout()) {
-    return FormOfWay::ROUNDABOUT;
-  } else if (use == TripLeg::kRampUse || use == TripLeg::kTurnChannelUse) {
-    return FormOfWay::SLIPROAD;
-  } else if (rc == RoadClass::kMotorway && one_way) {
-    return FormOfWay::MOTORWAY;
-  } else if (rc <= RoadClass::kTertiary && one_way) {
```

Comment:
i dont think the road class plays a role in whether a road is single or dual+ carriageway. the best way we can differentiate this is to say can you drive in both directions or can you drive in at least one direction